docs/root/operations/cli.rst

```diff
@@ -20,6 +21,14 @@ following are the command line options that Envoy supports.
   <https://developers.google.com/protocol-buffers/docs/reference/cpp/google.protobuf.text_format>`_
   formats respectively.
 
+.. option:: --config-json <json string>
+
+  *(optional)* The JSON string for a v1 of v2 bootstrap configuration. If this flag is missing,
```

Comment:
Can you potentially discuss escaping issues? Are there any? I'm not really clear how a config would be passed. A small example would be amazing.

Reply:
Done.